packages/trader/src/AppV2/Utils/trade-params-utils.tsx

```diff
@@ -42,3 +49,51 @@ export const isDigitContractWinning = (
     if (!contract_type || !win_conditions[contract_type]) return false;
     return win_conditions[contract_type];
 };
+
+export const getTradeTypeTabsList = (contract_type = '') => {
+    const is_turbos = isTurbosContract(contract_type);
+    const is_vanilla = isVanillaContract(contract_type);
+    const is_high_low = contract_type === TRADE_TYPES.HIGH_LOW;
+    const is_touch = isTouchContract(contract_type);
+    const tab_list = [
+        {
+            label: 'Up',
```

Comment:
Should these labels be localized?

Reply:
no, we don't localize contract types or their subtypes (no need for localize() that we have in V1), translation is not happening from translation team anyway.